src/components/Plans/PlanCalculator/PlanCalculatorButtons/index.js

```diff
@@ -50,11 +60,20 @@ export const PlanCalculatorButtons = InjectAppServices(
   },
 );
 
-const getBuyPurchaseUrl = ({ controlPanelUrl, planId, discountId, promoCode }) => {
-  return (
-    controlPanelUrl +
-    `/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=${planId}&fromStep1=True` +
-    `${discountId ? `&IdDiscountPlan=${discountId}` : ''}` +
-    `${promoCode ? `&PromoCode=${promoCode}` : ''}`
-  );
+const getBuyPurchaseUrl = ({
+  controlPanelUrl,
+  planType,
+  planId,
+  discountId,
+  promoCode,
+  newCheckoutEnabled,
+}) => {
+  return newCheckoutEnabled
+    ? `/checkout/premium/${planType}?selected-plan=${planId}` +
+        `${discountId ? `&IdDiscountPlan=${discountId}` : ''}` +
```

Comment:
@nreal ,  Could you please rename the parameter: ```IdDiscountPlan``` by ```discountId```?. This parameter's name is expected in the checkout page.

Reply:
Yes, it is ok done